dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task53800ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,462 @@
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(testBody)
+                .host(host)
+                .nextPersisted();
+
+
+        final Task53800ChangeContainerPathToAbsolute task53800ChangeContainerPathToAbsolute =
+                new Task53800ChangeContainerPathToAbsolute();
+
+        task53800ChangeContainerPathToAbsolute.executeUpgrade();
+
+        final ArrayList templates = getTemplateFromDataBase(template);
+
+        final HashMap templateMap = (HashMap) templates.get(0);
+        assertEquals(1, templates.size());
+
+        final String drawedBody = templateMap.get("drawed_body").toString();
+        assertEquals(layout, drawedBody);
+    }
+
+    /**
+     * Method to Test: {@link Task53800ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with relative path container in the drawed_body fields and the body field is null
+     * Should: Should turn it into a Absolute Path, using the template's host
+     */
+    @Test
+    public void whenTemplateLayoutHasRelativePathButBodyIsNullShouldTurnIntoAAbsolutePath() throws IOException, DotDataException {
+
+        final String layout = String.format(jsonDrawBody, "");
+        final Host host = new SiteDataGen().nextPersisted();
+
+        checkTemplateLayout(layout);
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .setBodyAsNull()
+                .host(host)
+                .nextPersisted();
+
+
+        final Task53800ChangeContainerPathToAbsolute task53800ChangeContainerPathToAbsolute =
+                new Task53800ChangeContainerPathToAbsolute();
+
+        task53800ChangeContainerPathToAbsolute.executeUpgrade();
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    private void checkTemplateFromDataBase(Host host, Template template) throws DotDataException {
+        final ArrayList templates = getTemplateFromDataBase(template);
+
+        final HashMap templateMap = (HashMap) templates.get(0);
+        assertEquals(1, templates.size());
+
+        final String drawedBody = templateMap.get("drawed_body").toString();
+        assertTrue(drawedBody.contains(String.format("//%s/application/containers/default/", host.getHostname())));
+
+        if (!Strings.isNullOrEmpty(templateMap.get("body").toString())) {
+            final String body = templateMap.get("body").toString();
+            assertTrue(body.contains(String.format("//%s/application/containers/default/", host.getHostname())));
+        }
+    }
+
+    private ArrayList getTemplateFromDataBase(Template template) throws DotDataException {
```

Comment:
Codacy found an issue: [Avoid using implementation types like 'ArrayList'; use the interface instead](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)